packages/react-components/package.json

```diff
@@ -34,6 +34,7 @@
     "@babel/preset-env": "^7.15.6",
     "@babel/preset-react": "^7.14.5",
     "@babel/preset-typescript": "^7.15.0",
+    "@livechat/design-system-material-icons": "^1.0.0",
```

Comment:
I would move it into `dependencies` because DS components depend on this package.

Similar to how it was done in "old" DS. The old "DS" has dependency on `material-ui-icons` ;)

Reply:
fixed